apps/backend/src/authn/github.strategy.ts

```diff
@@ -74,9 +75,13 @@ export class GithubStrategy extends PassportStrategy(Strategy, 'github') {
       .then((response): GithubProfile => {
         return response.data;
       });
-    const {firstName, lastName} = this.authnService.splitName(
-      userInfoResponse.name
-    );
+    let firstName = userInfoResponse.login;
+    let lastName = '';
+    if (typeof userInfoResponse.name === 'string') {
+      firstName = this.authnService.splitName(userInfoResponse.name).firstName;
+      lastName = this.authnService.splitName(userInfoResponse.name).lastName;
+    }
```

Comment:
Split with the all additional names going into the lastname field instead of first name field.

Try using a ternary operator so we can still use `const {firstName, lastName}` destructuring.